Patch origin: in-house
Patch status: Solaris-specific; not suitable for upstream

Fixes problem where APR-util Parfait build wouldn't analyze mysql and
sqlite3 modules as apr_dbd_sqlite3.o.bc and apr_dbd_mysql.o.bc were
removed here.

--- build/ltmain.sh	2013-11-13 08:14:27.000000000 -0800
+++ build/ltmain.sh	2014-04-09 02:30:03.737786743 -0700
@@ -7548,7 +7548,7 @@
 	tempremovelist=`$ECHO "$output_objdir/*"`
 	for p in $tempremovelist; do
 	  case $p in
-	    *.$objext | *.gcno)
+	    *.$objext | *.$objext.bc | *.gcno)
 	       ;;
 	    $output_objdir/$outputname | $output_objdir/$libname.* | $output_objdir/${libname}${release}.*)
 	       if test "X$precious_files_regex" != "X"; then
